src/main/java/it/gov/pagopa/pu/bff/connector/debt_position/client/DebtPositionClient.java

```diff
@@ -55,5 +58,15 @@ public DebtPositionDTO getDebtPosition(Long debtPositionId,
       return null;
     }
   }
+
+  public Boolean deleteDebtPosition(Long debtPositionId, String accessToken){
```

Comment:
let's add a javadoc to describe what's the meaning of true or false
```suggestion
  public boolean deleteDebtPosition(Long debtPositionId, String accessToken){
```